config/jobs/kubernetes-sigs/cli-utils/cli-utils-presubmit-master.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes-sigs/cli-utils:
+  - name: cli-utils-presubmit-master
+    decorate: true
+    always_run: true
+    path_alias: sigs.k8s.io/cli-utils
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: golang:1.13
+        command:
+        - make prow-presubmit-check
```

Comment:
I think they need to be in separate lines :-)

Reply:
done, thanks